parse/parser.go

```diff
@@ -346,6 +356,10 @@ func parseStatement(buf TokenBuffer) (ast.Statement, error) {
 		return parseIfStatement(buf)
 	case Return:
 		return parseReturnStatement(buf)
+	case Lbrace:
```

Comment:
I think we don't support 

```
func foo() {
  {
      int a = 1
  }
}
```
such statement. So we don't need `Lbrace` case now

Reply:
We may implement this after scoping problem solved.